### ISSUE TITLE
Binding of i18n not working in mocha rest call tests

### ISSUE THREAD

**Issue by rvan1** [mashpie/i18n-node#314]
I'm testing i18n in an arrow appcelerator app (express/mongo). I want to use it for response error messages and not in View templates. I can get translations in a validation process with something like 'i18n.__('MISSING_PARAMETER')', but when I run mocha REST API call tests, the translations fail and I just get the key (MISSING_PARAMETER). Each test method starts a new server so I have 'app.use' in each. Can the module be used the way I'm trying?

it.only('Create - fail - missing firstName', (done) => {
    var server = new Arrow({
      ignoreDuplicateModels: true
    })
    i18n.configure({
      defaultLocale: 'en',
      directory: __dirname + '/i18n',
      register: global
    })
    server.on('starting', function () {
      server.app.use(i18n.init) 
    })
    // start server
    server.start((err, data) => {
      if (err) {
         .. test rest call...

**Comment by rvan1** [comment]
My issue was just that the translation jsons weren't found on the 'directory' path, which is something I had suspected and tried to troubleshoot via trial and error, then was able to fix after tracing through the code and logging out the path in scope during mocha testing. When file isn't found the locales, at least for me, got loaded with the only key I had added a lookup for, and it was both key and value, and was listed twice. No other translations got loaded. So the result was confusing and I don't know where the logError and logDebug go, though I'm not sure there was actually an error.
